app/views/shared/_flash.html.erb

```diff
@@ -51,6 +51,6 @@
     class="flash-banner text-center hidden-xs"
     role="alert"
   >
-    <%= render :file => tidelift_flash_partial %>
+    <%== render :file => tidelift_flash_partial %>
```

Comment:
was this one a typo/debugging change?

Reply:
I couldn't track this back to a change in Rails, so this could be not the quite right fix, but after the upgrade I was getting the escaped html from this using `render :file`.  AFAICT this was all still correct usage

Reply:
interesting! The erubi docs confirm that it's the escaped version of `<%=`. TIL!